lib/content-types/merge.js

```diff
@@ -60,6 +67,13 @@ const squish = (types) => {
         plugin.id = attribute.id;
         plugin.type = attribute.type;
 
+        // Set Default Label from Plugin Name
+        if (Object.keys(plugin.inputs).length === 1) {
+          const input = Object.keys(plugin.inputs)[0];
+
+          plugin.inputs[input].label = plugin.name;
```

Comment:
Confused on this...if this was used on [the text plugin](https://github.com/punchcard-cms/input-plugin-text/blob/master/index.js), then wouldn't it change the `label` from "Add your text" to "Text"? Wouldn't we rather it was "Add your text"?

from input plugin:
```
module.exports = {
  name: 'Text',
  description: 'An input plugin for a single text file',
  validation: {
    textValidation: validation,
  },
  inputs: {
    text: {
      validation: {
        function: 'textValidation',
        on: 'blur',
      },
      label: 'Add Your Text',
```